coordinator/coordinator.go

```diff
@@ -621,25 +613,42 @@ func (c *Coordinator) runPrimaryTableJob(source *metadata.SourceVariant, resID m
 	if sourceName == "" {
 		return fferr.NewInvalidArgumentError(fmt.Errorf("source name is not set"))
 	}
+	err := c.Metadata.Tasks.AddRunLog(runInfo.TaskId, runInfo.RunId, "Starting Registration...")
+	if err != nil {
+		return err
+	}
 	if _, err := offlineStore.RegisterPrimaryFromSourceTable(providerResourceID, sourceName); err != nil {
 		return err
```

Comment:
should the errors be added to runlogs?

Reply:
Errors are added at a higher level when anything returns